keps/sig-apps/3998-job-success-completion-policy/README.md

```diff
@@ -381,214 +382,6 @@ To avoid the above conflicts, terminating policies are evaluated the first befor
 This means that the terminating policies are respected rather than the successPolicies,
 if the Job doesn't have the `FailureTarget` or `SuccessCriteriaMet` conditions yet.
 
```

Comment:
In the PRR section of this document you need to fill in the missing section`###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?`

Reply:
Also make sure to update the testing section with links to both integration and e2e tests added during alpha stage.

Reply:
The whole section `#### Optional Second Alpha` can also be removed, now that you're moving this as potential future work.

Reply:
@soltysh I addressed all comments. PTAL. Thanks!